packages/jsii-diff/bin/jsii-diff.ts

```diff
@@ -4,6 +4,8 @@ import * as reflect from 'jsii-reflect';
 import * as log4js from 'log4js';
 import * as yargs from 'yargs';
 
+import '@jsii/node-check';
```

Comment:
I wonder if having `@jsii/node-check` be a script that just runs on import vs a function that needs to be called explicitly is the way to go? Since this is a separate module that is consumed by various CLIs and the runtime it seems like we should let those consumers decide where exactly this runs in their lifecycle?

Reply:
I was trying to go for a boilerplate-free approach as much as possible. Import & forget is a pretty simple way to go... but I can actually offer both...

Have it export a check function... And have an import-and-run name like `@jsii/node-check/run` (although I don't love that particular name).